lib/beacon/loader/page_module_loader.ex

```diff
@@ -64,7 +64,7 @@ defmodule Beacon.Loader.PageModuleLoader do
       end
 
       """
-        def handle_event(#{path_to_args(path, "")}, #{event.event_name}, event_params, socket) do
+        def handle_event(#{path_to_args(path, "")}, "#{event.event_name}", event_params, socket) do
```

Comment:
@TheFirstAvenger fixing the event pattern match in here. I guess it was supposed to match in the string `event_name`, right? Let me know if I'm missing something.

Reply:
Yes, good catch!